loopy/isl_helpers.py

```diff
@@ -83,6 +83,13 @@ def make_slab(space, iname, start, stop, step=1):
 
     :arg step:
         An instance of :class:`int`.
+
+
+    .. note::
+
+        Not to be confused with the access map of the slice
+        ``[start:stop:step]``. This routine simply creates a basic set as
+        advertised above.
```

Comment:
```suggestion
        Despite the use of *start*, *stop*, *step* arguments,
        this function does *not* return a :class:`~islpy.BasicSet`
        of indices that might be accessed via 
        `array[slice(start, stop, step)]`.
        Instead, it returns an index that can be used as
        ``start + iname * step``.
        ``[start:stop:step]``.
```
Note that you mention `start <= iname *step`, which is different from what I suggest here. I don't think `start <= iname *step` is correct, as for `slice`, the base index is always included.

Reply:
The name `step` is quite misleading in the context of make_slab. Let me just rename it to `iname_multiplier`.

Reply:
Let me know if you disagree with this.

Reply:
Didn't apply this as post-9c1b2ba there shouldn't be a confusion between make_slab and a slice's access map.